tests/mirror_test.go

```diff
@@ -78,7 +79,7 @@ var testMirrorFunc = func(version string) {
 						Eventually(func() error {
 							_, err := clusterApi.ReadFileFromPod(podConsName, "test", "/tcpdump.log")
 							return err
-						}, 120 * time.Second, time.Second).Should(Succeed(), "tcpdump did not start in time");
+						}, 120*time.Second, time.Second).Should(Succeed(), "tcpdump did not start in time")
```

Comment:
IDE kept removing these spaces around "*" LMK if you prefer i edit the file in vim or smthn and remove this change.

Reply:
As long as it passes the CI it's all right